test/recommended.js

```diff
@@ -85,16 +85,16 @@ describe('recommended config', () => {
             }
         `);
 
-        const { messages } = report.results[0];
+        const { messages } = results[0];
         assert.strictEqual(messages.length, 2);
-        assert.strictEqual(messages[0].ruleId, '@lwc/lwc/no-dupe-class-members');
-        assert.strictEqual(messages[1].ruleId, '@lwc/lwc/no-dupe-class-members');
+        assert.strictEqual(messages[0].ruleId, 'no-dupe-class-members');
+        assert.strictEqual(messages[1].ruleId, 'no-dupe-class-members');
```

Comment:
Looks like there is now an official ESLint rule: https://eslint.org/docs/rules/no-dupe-class-members

Reply:
We should deprecate the `@lwc/lwc/no-dupe-class-members` rule now that ESLint has builtin support for this.

Reply:
It looks like `@lwc/lwc/no-dupe-class-members` was [designed](https://github.com/salesforce/eslint-plugin-lwc/blob/master/docs/rules/no-dupe-class-members.md) to extend the built-in rule which lacked support for class fields. It appears to have support for class fields now. So yeah, it can be removed.